modules/aws-rds-cluster-instance/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "this" {
+  description = "Aurora MySQL Cluster instance"
```

Comment:
Since it's a generic module, let's not mention "MySQL"

Reply:
Definitely, nice catch!